src/runtimes/web/default_strategy.ts

```diff
@@ -17,16 +17,16 @@ var getDefaultStrategy = function(config : any) : any {
 
   return [
     [":def", "ws_options", {
-      hostUnencrypted: config.wsHost + ":" + config.wsPort,
-      hostEncrypted: config.wsHost + ":" + config.wssPort,
+      hostNonTLS: config.wsHost + ":" + config.wsPort,
+      hostTLS: config.wsHost + ":" + config.wssPort,
       httpPath: config.wsPath
     }],
     [":def", "wss_options", [":extend", ":ws_options", {
-      encrypted: true
+      useTLS: true
     }]],
     [":def", "sockjs_options", {
-      hostUnencrypted: config.httpHost + ":" + config.httpPort,
-      hostEncrypted: config.httpHost + ":" + config.httpsPort,
+      hostNonTLS: config.httpHost + ":" + config.httpPort,
```

Comment:
From what I can tell this is a breaking API change for customers who were using the hostEncrypted/hostUnencrypted options - was this intentional?

Reply:
This is ok actually, the public config options are https://github.com/pusher/pusher-js#wshost-wsport-wssport-httphost-httpport-httpsport, and don't need to change.

`hostNonTLS` and `hostTLS` are purely internal.